src/CMakeLists.txt

```diff
@@ -40,6 +40,8 @@ else()
     endif()
 endif()
 
+set(ENABLE_GMOCK ON CACHE STRING "")
```

Comment:
I don't think users should be able to selectively disable gmock. They should be able to disable testing in general, but why allow them to disable gmock? I think this would just complicate things without adding anything of value. If a user tries to disable it while enabling testing, I think build errors are a reasonable outcome.